msal/src/Microsoft.Identity.Client/Platforms/uap/PlatformInformation.cs

```diff
@@ -72,32 +74,8 @@ public override string GetDeviceModel()
 
         public override async Task<bool> IsUserLocalAsync(RequestContext requestContext)
         {
-            if (!UserInformation.NameAccessAllowed)
-            {
-                // The access is not allowed and we cannot determine whether this is a local user or not. So, we do NOT add form auth parameter.
-                // This is the case where we can advise customers to add extra query parameter if they want.
-
-                const string msg =
-                    "Cannot access user information to determine whether it is a local user or not due to machine's privacy setting.";
-                requestContext.Logger.Info(msg);
-                requestContext.Logger.InfoPii(msg);
-                return false;
-            }
-
-            try
-            {
-                return string.IsNullOrEmpty(await UserInformation.GetDomainNameAsync().AsTask().ConfigureAwait(false));
-            }
-            catch (UnauthorizedAccessException ae)
-            {
-                requestContext.Logger.Warning(ae.Message);
-                const string msg = "Cannot try Windows Integrated Auth due to lack of Enterprise capability.";
-                requestContext.Logger.Info(msg);
-                requestContext.Logger.InfoPii(msg);
-                // This mostly means Enterprise capability is missing, so WIA cannot be used and
-                // we return true to add form auth parameter in the caller.
-                return true;
-            }
+            IReadOnlyList<Windows.System.User> users = await Windows.System.User.FindAllAsync();
```

Comment:
>IReadOnlyList<Windows.System.User> users = await Windows.System.User.FindAllAsync(); [](start = 12, length = 84)

minor: change we are no longer telling in the log that there is no WIA users available (local)